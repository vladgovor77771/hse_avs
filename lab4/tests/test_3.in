4
qxyz
3
10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 20 21
ouyt
3
10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 20 21
mnkj
3
10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 20 21
abcd
3
10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 10 11 12 13 14 15 16 17 18 19 20 21
