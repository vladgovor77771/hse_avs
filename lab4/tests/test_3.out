abcd => 10 11 12 13 
mnkj => 12 13 10 19 
ouyt => 14 10 14 19 
qxyz => 16 13 14 15 
